README.md

```diff
@@ -120,6 +120,15 @@ ResponseDumper.define 'Users' do
 end
 ```
 
+## Running Dumps for Specific Files
```

Comment:
Up to you, but I think this should go directly under "usage" as it seems morefundamental than before/after hooks.